test/lib/creation.go

```diff
@@ -322,6 +322,17 @@ func (c *Client) CreateSinkBindingV1Alpha2OrFail(sb *sourcesv1alpha2.SinkBinding
 	c.Tracker.AddObj(sb)
 }
 
+// CreateSinkBindingV1Beta1OrFail will create a SinkBinding or fail the test if there is an error.
+func (c *Client) CreateSinkBindingV1Beta1OrFail(sb *sourcesv1beta1.SinkBinding) {
```

Comment:
nit: based on the convention in eventing e2e test is, naming should be `CreateSinkBindingV1beta1OrFail